core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -351,25 +351,41 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
     }
   }
 
-  "Expressions in CTE" in {
-    """WITH
-      |    a AS (1),
-      |    b AS (2),
-      |    t (d, e) AS (SELECT 4, 5),
-      |    c AS (3)
-      |SELECT
-      |    a + b,
-      |    a * c,
-      |    a * t.d
-      |FROM t;""".stripMargin transpilesTo
+  "Common Table Expressions (CTEs)" should {
+    """support expressions""" in {
```

Comment:
would "have precedence" be better ?

Reply:
I think this comment was intended for the other test, and I appreciate the suggestion: the language was clumsy and I've fixed it in 63eb9bfacc656502d6209da2a58527a765086214.